server/controllers/rescue.ctrl.js

```diff
@@ -15,4 +17,37 @@ module.exports = {
         res.send(err);
       });
   },
+
+  // Create new rescue entry
+  addNewRescue: (req, res, next, io) => {
+    const rescue = req.body;
+    const {
+      contactPerson, contactNumber, location, noOfPerson,
+    } = rescue;
+
+    if (
+      isEmpty(contactPerson)
+      || isEmpty(contactNumber)
+      || isEmpty(location)
+      || isEmpty(noOfPerson)
+      || (isEmpty(location.address) && (isEmpty(location.lat) || isEmpty(location.lon)))
```

Comment:
Also add additionalInfo.
Currently in the chatbot it's location.address or location.lat and long so with this line adding a new rescue would always fail if it is from the bot.

Reply:
(1) The if block is only used to check missing required fields - additionalInfo is an optional field. 
(2) For the location condition, it would only return an error if location.address is empty at the same time one of the location.lon or location.lat is also empty. Else should be valid